RELEASE_NOTES.md

```diff
@@ -6,6 +6,16 @@ v1.0.0-alpha.X
 
 ### Breaking changes
 
+- Redesigned `managementPolicy` to return traits rather than a bitfield
```

Comment:
[fix] Just as we have a `TraitSet` and `TraitsData`...
```suggestion
- Redesigned `managementPolicy` to return a `TraitsData` rather than a bitfield
```

Reply:
Updated in c92fb7cd